test/e2e/tests/test_replicationgroup.py

```diff
@@ -607,8 +606,8 @@ def test_rg_fault_tolerance(self, rg_fault_tolerance):
                 raise AssertionError(f"Unknown node {node['cacheClusterID']}")
 
         # assert AF and multi AZ
-        assert resource['status']['automaticFailover'] == "disabled"
-        assert resource['status']['multiAZ'] == "disabled"
+        assert resource['status']['automaticFailover'] == "enabled"
```

Comment:
these were just wrong assertions; if you check the file the last patch enables both of these fields